_data/education.yml

```diff
@@ -392,6 +392,12 @@ websites:
     facebook: ScottishQualificationsAuthority
     img: mysqa.png
 
+  - name: Open University
+    url: https://www.open.ac.uk/
+    twitter: OpenUniversity
+    facebook: theopenuniversity
+    img: openuniversity.png
+
```

Comment:
```suggestion
    regions:
      - uk

```

Reply:
As per ticket:

"Please note that because this is an accredited UK university, but specialises in remote & distance learning, it is region-less."